frontends/election-manager/src/screens/unconfigured_screen.tsx

```diff
@@ -209,29 +214,14 @@ export function UnconfiguredScreen(): JSX.Element {
     void updateStatus();
   }, [updateStatus]);
 
-  if (isUploading) {
+  if (isUploading || isLoading) {
     return (
       <NavigationScreen>
```

Comment:
Add the `centerContent` prop here to center the “Loading…” text in the content area.